docs/source/standard.md

```diff
@@ -255,39 +255,41 @@ The ability of the organisation to support users who want to browse the data ava
+The ability of the organisation to deploy and manage encryption to protect information assets, including data for TRE research projects.
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement                                             | Guidance                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ----------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TREs can encrypt data as an extra layer of protection | Encryption can be seen as an extra layer of protection for sensitive data already protected by the security controls of a TRE. This prevents unauthorised access to the data even if the storage media is compromised. Approved researchers can be granted access to the mechanism to decrpyt the data before using it in their research.                                                                         |
+| TREs should encrypt data when in transit              | TRE architectures that are built such that data gets transmitted over networks (between machines), should ensure data is encrypted in transit. This safeguards the data from interception or tampering during transmission.                                                                                                                                                                                       |
+| Encryption software should be updated                 | The latest security patches and updates should be applied to any enctyption software being used by the TRE. This helps address any known vulnerabilities or weaknesses in the encryption implementation.                                                                                                                                                                                                          |
+| TREs should use secure key management                 | TREs should employ secure key management practices, including storing encryption keys separately from the encrypted data and implementing strong access controls (e.g. Single Sign On) for key management systems. For example, you could use a cloud-based solution such as Azure Key Vault to manage all encryption keys and administrator keys to TRE infrastructure such as (virtual) machines and databases. |
```

Comment:
Worth discussing user-managed vs. platform-managed encryption or is that too cloud-dependent?

Reply:
I think we might _have_ to in order to be clear about what we are requiring/suggesting.

For example, I would say that platform-managed encryption is similar to users' home directories being encrypted with their password as the passphrase. It protects you from an external bad actor _e.g._ someone steals the storage. But for authorised people the encryption is transparent and there is no extra protection here from internal bad actors.

See my review comment for some specific ideas.